EnvChanger/Classes/EnvChangerController.swift

```diff
@@ -174,30 +178,23 @@ public class EnvChangerController<T>: UIViewController where T: EnvironmentRepre
         let view = UIView()
         let button = UIButton(type: .custom)
         
-        DispatchQueue.main.async { [weak self] in
-            guard let buttonConfiguration = self?.buttonConfiguration else { return }
-            
-            switch buttonConfiguration {
-            case .title(let title):
-                button.setTitle(title, for: .normal)
-                button.titleLabel?.font = .systemFont(ofSize: 12)
-                
-                button.layer.cornerRadius = 4
-                button.layer.borderWidth = 1
-                button.setTitleColor(.white, for: .normal)
-                button.backgroundColor = .gray
-                return
-            case .image(let image):
-                button.setImage(image, for: .normal)
-                button.imageEdgeInsets = UIEdgeInsets(top: 30, left: 30, bottom: 30, right: 30)
-                button.imageView?.contentMode = .scaleAspectFit
-            }
+        switch buttonConfiguration.style {
+        case .title(let title):
+            button.setTitle(title, for: .normal)
```

Comment:
Both customisations for the button can be moved as extension methods and constants can be separately defined.

Reply:
Done.